src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/ConfigurationClient.cs

```diff
@@ -17,6 +17,10 @@ public partial class ConfigurationClient
         private readonly Uri _baseUri;
         private readonly HttpPipeline _pipeline;
 
+        protected ConfigurationClient()
```

Comment:
Can you add doc comments saying that this is for mocking scenarios

Reply:
Will do.